config/config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   test: {
     username: process.env.DB_USERNAME,
     password: process.env.DB_PASSWORD,
-    database: "koolskool_testdb",
+    database: process.env.DB_DATABASE,
```

Comment:
Bryson - I'm not liking this change.  Lets keep it Koolskool_testdb so that we don't accidentally overwrite our database.